OpenUpSA/wazimap-ng-ui
# re-selecting the current filter bug

## Description
re-selecting a current filter results in filters being shown but not applied

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-161

## How is it tested automatically?
added test steps into `__tests__/gui/point_data.feature`

## How should a reviewer test it locally
* select any indicator & subindicator from data mapper
* filter choropleth by selecting a group and a value
* select the same group and the same value again
* confirm that the choropleth is filtered again

## Screenshots
![ezgif-3-7e497c264f29](https://user-images.githubusercontent.com/53019884/142739514-354b3b20-958a-4147-9bdb-f3aa4b3894cf.gif)

## Changelog

### Added

### Updated
* `src/js/ui_components/filter_row.js` to fix the issue

### Removed
* `src/js/elements/subindicator_filter/subindicator_filter.js` the file is not used anymore

## Checklist


### Pull Request


### Code Quality


### Testing
